_topic_maps/_topic_map_rosa.yml

```diff
@@ -607,3 +607,4 @@ Topics:
   File: rosa-troubleshooting-deployments
 - Name: Red Hat OpenShift Service on AWS managed resources
   File: rosa-managed-resources
+---
```

Comment:
```suggestion

```

This can't be `---`. It needs to just be an internal EOF marker, or perhaps a blank line.

Reply:
Made the changes